src/test/e2e/services/group-service.e2e.test.ts

```diff
@@ -69,3 +72,28 @@ test('should add person to completely new group with new name', async () => {
     expect(groups.length).toBe(1);
     expect(groups[0].name).toEqual('dev_group');
 });
+
+test('should not update groups when not string array ', async () => {
+    await groupService.syncExternalGroups(user.id, 'Everyone' as any);
+    const groups = await groupService.getGroupsForUser(user.id);
+    expect(groups.length).toBe(1);
```

Comment:
`expect(groups).toHaveLength(1)` if you want it to flow better. It does exactly the same behind the scenes so I'm fine with either. Just thought it was nice to know of the API